### PR TITLE
UpperCamelCase getters and setters

### PR DESCRIPTION
**Pull request by jderusse**
Fixes https://github.com/async-aws/aws/pull/537#discussion_r420435886

### CODE REVIEW

**Comment by Nyholm** [comment]
This is fine since php method names are not case sensitive.

**Comment by jderusse** [comment]
Indeed, I wonder if we shouldn't lower the property name for consistency?

**Comment by Nyholm** [comment]
Na. I don’t think we should change the properties. There is no real win and it will make the generators more complicated.

File: src/CodeGenerator/src/Generator/PaginationGenerator.php [hunk 1]
```diff
@@ -114,7 +114,7 @@ private function generateOutputPagination(Operation $operation, Pagination $pagi
             [$returnType, $iteratorType] = $this->typeGenerator->getPhpType($listShape);
             $iteratorTypes[] = $iteratorType;
 
-            $getter = 'get' . $resultKey;
+            $getter = 'get' . \ucfirst($resultKey);
```
Comment (guillaumesmo) [inline src/CodeGenerator/src/Generator/PaginationGenerator.php:117]: this could be moved up, and be used in line 104 as well

**Comment by guillaumesmo** [comment]
I was planning to have a PR for this as well, but I slept on it and it's already done :)

awesome!

File: src/CodeGenerator/src/Generator/PaginationGenerator.php (hunk 1 above)
Comment (jderusse) [reply src/CodeGenerator/src/Generator/PaginationGenerator.php]: good catch, thanks

**Comment by jderusse** [comment]
thanks for the review

**Pull request merged**